src/debugger/Debugger.h

```diff
@@ -21,6 +21,7 @@
 #define __Debugger__
 
 #include "util/Vector.h"
+#include "api/EscargotPublic.h"
```

Comment:
`api/EscargotPublic.h` was first designed to expose Escargot API functions to other programs,
and Escargot's internal source does not have any dependence on this api file.
(exceptionally, `Template` has dependence but this case is inevitable)
So, I'm wondering that is it possible to implement debugger API without any internal dependence on the api file.

Reply:
I am trying to avoid duplication of types (classes, vectors, etc.), since both the external api and the debugger should use several common types. I am open to any idea how could we share types between the public api and the debugger implementation. Maybe creating an `"EscargotDebuggerPublic.h"` could help. Please let me know your suggestion.

Reply:
Another option could be to move some parts of the debugger to `EscargotPublic.cpp`. That would probably require some type duplication, but the `.h` dependency could be dropped.

Reply:
IMO, if possible, the second option looks good to me :)

Reply:
Are you going to remove `EscargotPublic.h` from this `Debugger.h` file?

Reply:
Yes! I am doing it now.